kernel-scriptlets/cert-script

```diff
@@ -21,9 +21,19 @@ while true ; do
     shift
 done
 
+is_efi () {
+    local msg rc=0
+
 # The below statement fails if mokutil isn't installed or UEFI is unsupported.
 # It doesn't fail if UEFI is available but secure boot is off.
-if ! mokutil --sb-state 2>/dev/null; then
```

Comment:
It would have been sufficient to replace this line by `mokutil --sb-state >/dev/null 2>&1`